src/components/Detail/Detail.scss

```diff
@@ -1,21 +1,9 @@
-@import url('https://fonts.googleapis.com/css?family=Lato:400,700');
-$font: 'Lato', sans-serif;
-
-$white: #F5F5F5;
-$light: #E0C9CB;
-$tan: #C3A1A0;
-$rose: #BA7E7E;
-$dark: #4E4E4E;
-$nav-color: #2E3B55;
-
-$breakpoint-tablet: 850px;
-$breakpoint-mobile: 600px;
+@import '../../global/constants.scss';
```

Comment:
better to use absolute (not relative) import paths, for example, in this case:
`@import 'global/constants.scss`